app/scripts/modules/amazon/src/serverGroup/details/rollback/rollbackServerGroup.controller.js

```diff
@@ -18,6 +18,31 @@ module.exports = angular.module('spinnaker.amazon.serverGroup.details.rollback.c
 
       var desired = serverGroup.capacity.desired;
 
+      var rollbackType = 'EXPLICIT';
+
+      if (allServerGroups.length === 0 && serverGroup.entityTags) {
+        var creationMetadata = serverGroup.entityTags.creationMetadata;
+
+        if (creationMetadata.value.previousServerGroup) {
```

Comment:
worth adding a guard here to verify `creationMetadata` exists

Reply:
Gotcha. Incoming.